frontend/system/src/Admin/EditEntity/EditEntityView.tsx

```diff
@@ -10,12 +10,13 @@ export const EditEntityView = () => {
   const { code } = useParams();
   const { setSuffix } = useBreadcrumbs();
 
-  const { data: entity } = useEntity(code || '');
+  const { data: entity, isFetchedAfterMount } = useEntity(code || '');
```

Comment:
Haven't seen isFetchedAfterMount before.

Reply:
Yeah this was a bit of a weird one, but it was loading the edit forms with stale data after the approved pending change. It was immediately going and fetching the new data, like it new the data was invalid, but the form rendered with the initial stale stuff... so this waits until the refetch is complete 🤷‍♀️